builtin/credential/ldap/backend.go

```diff
@@ -206,13 +154,11 @@ func (b *backend) Login(ctx context.Context, req *logical.Request, username stri
 		return username, policies, ldapResponse, allGroups, nil
 	}
 
-	entityAliasAttribute, err := ldapClient.GetUserAliasAttributeValue(cfg.ConfigEntry, c, username)
-	if err != nil {
-		return "", nil, logical.ErrorResponse(err.Error()), nil, nil
-	}
-	if entityAliasAttribute == "" {
```

Comment:
The implementation of this can be found [here](https://github.com/hashicorp/vault/blob/be83f12fff085b897638900802cdde190279c08b/sdk/helper/ldaputil/client.go#L256).

I believe it boils down to finding the 'CN' (common name) attribute.